source-formatter.properties

```diff
@@ -618,27 +618,27 @@
         org.apache.ant:ant:1.10.11,\
         org.apache.commons:commons-collections4:4.4,\
         org.apache.commons:commons-compress:1.26.0,\
-        org.apache.cxf:cxf-core:3.4.10,\
-        org.apache.cxf:cxf-rt-bindings-soap:3.4.10,\
-        org.apache.cxf:cxf-rt-bindings-xml:3.4.10,\
-        org.apache.cxf:cxf-rt-databinding-jaxb:3.4.10,\
-        org.apache.cxf:cxf-rt-frontend-jaxrs:3.4.10,\
-        org.apache.cxf:cxf-rt-frontend-jaxws:3.4.10,\
-        org.apache.cxf:cxf-rt-frontend-simple:3.4.10,\
-        org.apache.cxf:cxf-rt-rs-client:3.4.10,\
-        org.apache.cxf:cxf-rt-rs-extension-providers:3.4.10,\
-        org.apache.cxf:cxf-rt-rs-json-basic:3.4.10,\
-        org.apache.cxf:cxf-rt-rs-security-jose:3.4.10,\
-        org.apache.cxf:cxf-rt-rs-security-jose-jaxrs:3.4.10,\
-        org.apache.cxf:cxf-rt-rs-security-oauth2:3.4.10,\
-        org.apache.cxf:cxf-rt-rs-sse:3.4.10,\
-        org.apache.cxf:cxf-rt-security:3.4.10,\
-        org.apache.cxf:cxf-rt-transports-http:3.4.10,\
-        org.apache.cxf:cxf-rt-ws-addr:3.4.10,\
-        org.apache.cxf:cxf-rt-ws-policy:3.4.10,\
-        org.apache.cxf:cxf-rt-wsdl:3.4.10,\
-        org.apache.cxf:cxf-tools-common:3.4.10,\
-        org.apache.cxf:cxf-tools-validator:3.4.10,\
+        org.apache.cxf:cxf-core:3.5.9,\
```

Comment:
Hi @jgarcialr ,

I noticed some tiny typo in the commit message, we may need to upgrade it to "LPD-33706 upgrade org.apache.cxf dependencies to 3.5.9 in source-formatter.properties"

Thanks,
Jeff